### ISSUE TITLE
EJSONPARSE issue

### ISSUE THREAD

**Issue by Kremzion** [renesansz/discord-greeter-bot#212]
so when I tried to execute dependencies, this is what the error says:

C:\Users\Kremzion\Desktop\Kremzbot>npm install discord.io winston --save
npm ERR! file C:\Users\Kremzion\Desktop\Kremzbot\package.json
npm ERR! code EJSONPARSE
npm ERR! JSON.parse Failed to parse json
npm ERR! JSON.parse Unexpected token “ in JSON at position 3 while parsing near '{
npm ERR! JSON.parse “name”: “greeter-bot...'
npm ERR! JSON.parse Failed to parse package.json data.
npm ERR! JSON.parse package.json must be actual JSON, not just JavaScript.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Kremzion\AppData\Roaming\npm-cache\_logs\2019-03-07T10_25_33_635Z-debug.log

I don't see any differences in "package.json" and in the sample given on the blog.
Please reply as soon as possible, thank you.

**Comment by gand988** [comment]
you will need to rewrite all the " in the package.json file and will work.

**Comment by ayetofu** [comment]
@gand988  im having the same issue and you explanation didn't help much can you explain differently please?

**Comment by gand988** [comment]
TL;DR You just need to replace all the quotation mark. 

@ayetofu ok, 
the error says "npm ERR! JSON.parse package.json must be actual JSON, not just JavaScript."

If you just copy-paste the "package.json" code from a website, there is a chance that the quotation marks are left/right double quotation mark, because how they wrote the article. 

here's the difference 
error -> “name”
ok ->     "name"

Hope to be more clear.

**Comment by ayetofu** [comment]
@gand988  it did thank you! can i message you on discord for further help?

**Comment by gand988** [comment]
> @gand988 it did thank you! can i message you on discord for further help?

check my portfolio and DM me on twitter

**Comment by HarryZombie** [comment]
@gand988 I tried to do what you said and it literally did nothing, can I have some help please?
